xml/appendix_values_yaml.xml

```diff
@@ -0,0 +1,1791 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE appendix
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<appendix xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="app.kvm">
+ <title>Appendix</title>
+ <info>
+      <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+        <dm:bugtracker>
+          </dm:bugtracker>
+      </dm:docmanager>
+    </info>
+    
+    <para>
+        This appendix contains copies of the complete 
+        <filename>values.yaml</filename> files for the SCF and UAA deployments.
+    </para>
+    
+ <sect1 xml:id="app.scf.values.yaml">
+  <title>Complete &scf; values.yaml file</title> 
+
+<para>
+    This is the &scf; <filename>values.yaml</filename> configuration
+    file that is shipped with the &helm; charts. It provides a complete reference
+    of configuration options for &scf;, and their default settings.
+</para>
+
+<screen>
+---
+env:
+  # List of domains (including scheme) from which Cross-Origin requests will be
+  # accepted, a * can be used as a wildcard for any part of a domain.
+  ALLOWED_CORS_DOMAINS: "[]"
+
+  # Allow users to change the value of the app-level allow_ssh attribute.
+  ALLOW_APP_SSH_ACCESS: "true"
+
+  # Extra token expiry time while uploading big apps, in seconds.
+  APP_TOKEN_UPLOAD_GRACE_PERIOD: "1200"
+
+  # List of allow / deny rules for the blobstore internal server. Will be
+  # followed by 'deny all'. Each entry must be follow by a semicolon.
+  BLOBSTORE_ACCESS_RULES: "allow 10.0.0.0/8; allow 172.16.0.0/12; allow 192.168.0.0/16;"
+
+  # Maximal allowed file size for upload to blobstore, in megabytes.
+  BLOBSTORE_MAX_UPLOAD_SIZE: "5000"
+
+  # The set of CAT test suites to run. If not specified it falls back to a
+  # hardwired set of suites.
+  CATS_SUITES: ~
```

Comment:
In case anybody asks: This is not customer-relevant (it's used for running internal test suites). It's just sort of painful to get rid of it in the released chart :)

Reply:
ok I'll remove it.

Reply:
@mook-as Though it will still be there-- what do you think off adding a comment to that effect, that it's for internal test suites?

Reply:
see https://github.com/SUSE/doc-cap/commit/0f2915eeff97893efeb6166883512fb52dcf4aec